install/restart_scripts/renew_ca_cert.in

```diff
@@ -105,23 +104,6 @@ def _main():
                     "Updating trust on certificate %s failed in %s" %
                     (nickname, db.secdir))
         elif nickname == 'caSigningCert cert-pki-ca':
-            # Update CS.cfg
-            cfg_path = paths.CA_CS_CFG_PATH
-            config = directivesetter.get_directive(
-                cfg_path, 'subsystem.select', '=')
-            if config == 'New':
-                syslog.syslog(syslog.LOG_NOTICE, "Updating CS.cfg")
-                if cert.is_self_signed():
-                    directivesetter.set_directive(
-                        cfg_path, 'hierarchy.select', 'Root',
-                        quotes=False, separator='=')
-                else:
-                    directivesetter.set_directive(
-                        cfg_path, 'hierarchy.select', 'Subordinate',
-                        quotes=False, separator='=')
-            else:
-                syslog.syslog(syslog.LOG_NOTICE, "Not updating CS.cfg")
-
```

Comment:
This change is from upstream ticket https://pagure.io/freeipa/issue/3737 which brought many changes, including tools like ipa-cacert-manage. We don't use subsystem.select anywhere else within IPA so I think this is safe to drop as long as PKI isn't either. It relied on subsystem.select to determine whether to set this at all, and only on the original server. This assumed that the initial CA server was never lost.